DOCS/man/options.rst

```diff
@@ -1652,17 +1652,34 @@ Video
     ``--keepaspect=no`` is used.
 
 ``--video-align-x=<-1-1>``, ``--video-align-y=<-1-1>``
-    Moves the video rectangle within the black borders, which are usually added
-    to pad the video to screen if video and screen aspect ratios are different.
-    ``--video-align-y=-1`` would move the video to the top of the screen
-    (leaving a border only on the bottom), a value of ``0`` centers it
-    (default), and a value of ``1`` would put the video at the bottom of the
-    screen.
+    When the video is bigger than the window, these move the displayed rectangle
+    to show different parts of the video. ``--video-align-y=-1`` would display
+    the top of the video, ``0`` would display the center (default), and ``1``
+    would display the bottom.
+
+    When the video is smaller than the window and ``--video-recenter`` is
+    disabled, these move the video rectangle within the black borders, which are
+    usually added to pad the video to the window if video and window aspect
+    ratios are different. ``--video-align-y=-1`` would move the video to the top
+    of the window (leaving a border only on the bottom), ``0`` would center it,
+    and ``1`` would put the video at the bottom of the window.
 
     If video and screen aspect match perfectly, these options do nothing.
 
+    Unlike ``--video-pan-x`` and ``--video-pan-y``, these don't go beyond the
+    video's boundaries or make the displayed rectangle drift off after zooming.
+
     This option is disabled if ``--keepaspect=no`` is used.
 
+``--video-recenter=<yes|no>``
```

Comment:
```suggestion
``--video-align-center=<yes|no>``
```

I don't have good name for this, without making it too long, but `recenter` suggest is it a "command" and not an option. It should be a state.

Reply:
Plenty of options are verbs though? align-center just sounds wrong.

Reply:
I don't understand what are we REcentering. If I zoom out, I keep the video center, not recenter, which implies previous uncentered state.

Reply:
Well recentering really refers to displayed video rectangle and not to the video itself, same as video-pan and video-align, they move the output rectangle especially when zoomed in, video is just the shared prefix of these options. So if the video rectangle was aligned to the left, it gets realigned to the center after zooming out. But I can see how it can be confusing so we can go with `--video-keep-centered` if you prefer?